src/types/index.ts

```diff
@@ -1,43 +1,10 @@
 /* istanbul ignore file */
 
-import { ApiOptions } from '@polkadot/api/types';
-import { TypeDef } from '@polkadot/types/types';
-import BigNumber from 'bignumber.js';
-
-import {
-  CorporateActionTargets,
-  DividendDistributionDetails,
-  OfferingDetails,
-  ScheduleDetails,
-  SubsidyData,
-  TaxWithholding,
-} from '~/api/entities/types';
-import { CreateTransactionBatchParams } from '~/api/procedures/types';
 import { CountryCode, ModuleName, TxTag, TxTags } from '~/generated/types';
-import {
-  Account,
-  BaseAsset,
-  Checkpoint,
-  CheckpointSchedule,
-  CustomPermissionGroup,
-  DefaultPortfolio,
-  DefaultTrustedClaimIssuer,
-  DividendDistribution,
-  FungibleAsset,
-  Identity,
-  Instruction,
-  KnownPermissionGroup,
-  Nft,
-  NftCollection,
-  NumberedPortfolio,
-  Offering,
-  PolymeshTransaction,
-  PolymeshTransactionBatch,
-} from '~/internal';
-import { Modify } from '~/types/utils';
 
```

Comment:
Should this get `export * from "module/types." files?

If someone was importing types it would probably be from this file, so moving them without a reexport could be a breaking change.

Reply:
Already being exported transitively.